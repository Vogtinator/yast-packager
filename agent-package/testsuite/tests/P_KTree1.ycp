{
return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"german","common.pkd":"baum9.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return SCR::Execute(.setInstallSelection,["c1","b1","c2-1","d3"], false);
}
{
    return SCR::Read(.dependencies);
}
