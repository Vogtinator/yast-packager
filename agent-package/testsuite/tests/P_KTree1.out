(true)
(true)
($["ADD":["c1-1", "c1-2", "c1-3", "c2-1-1", "c2-1-1-1", "c2-1-1-1-1", "c2-1-1-1-1-1", "c2-1-1-1-1-1-1", "c2-1-1-1-1-1-1-E", "b1-1", "b1-2", "b1-2-1", "b1-2-2", "b1-3"], "CONFLICT":[], "OBSOLETE":[], "REQUIRE":[]])
